migrations/sql/V2021.02.08_0852__ReportRun.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE orderlyweb_report_run
+(
+    key         TEXT      NOT NULL,
+    email       TEXT      NOT NULL,
+    date        TIMESTAMP NOT NULL,
+    report_name TEXT      NOT NULL,
+    instances   TEXT      NOT NULL,
+    params      TEXT      NOT NULL,
+    git_branch  TEXT,
+    git_commit  TEXT,
+    status      TEXT,
+    logs        TEXT,
+    report_id   TEXT,
```

Comment:
..and to name this column report_version

Reply:
Done - it was supposed to be `report_version_id` but again consistency should win